ci/Jenkinsfile.linux

```diff
@@ -0,0 +1,72 @@
+pipeline {
+  agent {
+    docker { 
+      label 'linux'
+      image 'statusteam/nim-status-client-build:latest'
+      /* allows jenkins use cat and mounts '/dev/fuse' for linuxdeployqt */
+      args '--entrypoint="" --cap-add SYS_ADMIN --security-opt apparmor:unconfined --device /dev/fuse'
```

Comment:
What does an empty entrypoint do?

Reply:
It allow Jenkins to use `cat` to pipe commands into the container. Bit of a hack.